### PR TITLE
feat(sbb-header): introduce active state

### LINKED ISSUE #3095: header action with current state
https://www.figma.com/design/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?node-id=29161-1064932&node-type=frame&t=l7cKnjSVzJWa9oAw-11

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #3095

### CODE REVIEW

File: src/elements/header/common/header-action.scss [hunk 1]
```diff
@@ -37,6 +41,9 @@
   @include sbb.if-forced-colors {
     --sbb-header-action-border-color: CanvasText;
     --sbb-header-action-color: LinkText;
+
+    // Hide border with forced colors, as state is displayed on border
+    --sbb-header-action-active-border-width: 0rem;
```
Comment (DavideMininni-Fincons) [inline src/elements/header/common/header-action.scss:46]: Is it possible to remove the 'rem' unit?

File: src/elements/header/common/header-action.scss [hunk 2]
```diff
@@ -155,7 +183,10 @@
       --_sbb-header-first-item-padding-shift: calc(
         -0.5 * (var(--sbb-header-action-min-width) - var(--sbb-header-action-icon-dimension))
       );
-      --sbb-header-action-padding-inline-zero: 0;
+      --sbb-header-action-padding-inline-zero: 0rem;
```
Comment (DavideMininni-Fincons) [inline src/elements/header/common/header-action.scss:186]: Is it possible to remove the 'rem' unit?

**Review by DavideMininni-Fincons — APPROVED** [review]
LGTM, i added just a small question.

File: src/elements/header/common/header-action.scss (hunk 2 above)
Comment (jeripeierSBB) [reply src/elements/header/common/header-action.scss:186]: not anymore. I used it differently first where it was needed :). Will remove it.

File: src/elements/header/common/header-action.scss (hunk 1 above)
Comment (jeripeierSBB) [reply src/elements/header/common/header-action.scss:46]: yes. Will do.

**Comment by mcilurzo** [comment]
LGTM. Thank you.

**Pull request merged**